### DISCUSSION TITLE
Goprp 8 Black can not update firmware Golab

### DISCUSSION THREAD

**Discussion by VuMrFPV** [gopro/labs#305] [Q&A]
Goprp 8 Black can not update firmware Golab
i copy Update to my sdCard and power on gopro
but in fornt screen show icon update,
i wait for long time it not successs.
![341475401_610329041018785_2672601688526497365_n](https://user-images.githubusercontent.com/130951180/232370495-95dc48de-4645-4a06-bd74-24dadbf39b16.jpg)

**Comment by dnewman-gpsw** [comment]
What version of firmware were you on before the update, and that may effect the way to solve this.

**Comment by Darhel** [comment]
Looks like I have same problem. Fw download icon is blinking, but nothing happening. 
Together with the icon blinks the frame on the edges of the LCD screen, if this fact is important, but perhaps it is just a decoration.
Looks like there is no way to check for current FW version, its GepRC Naked version with only front panel LCD screen that shows GoPro text by default and some tech info while recording. 
Any way to fetch current fw version?
Thank you.

**Reply by dnewman-gpsw** [reply]
On the SD card look in MISC/version.txt

```
{
"info version":"2.0"
,"firmware version":"H22.01.02.10.70"   <-  firmware version.
,"wifi mac":"2xxxxxxxxxx2"
,"camera type":"HERO11 Black"
,"camera serial number":"C3xxxxxxxxxxx"
}
```

If you are not running a recent version of Labs, it wasn't designed for a naked configuration, so I is looking for an authenticated battery.  You will like need to add the original battery cage and battery to complete the update.

**Reply by Darhel** [reply]
Cam bought as assembled by GepRC and working well with pre-installed Labs, I dont thing that they'd use any special flashing things, so it's must be compatible, but who knows. Actually I dont have any serious reasons to update it ;) But i'd like to use updated software where it's possible.

{
"info version":"2.0",
"firmware version":"HD8.01.02.50.71",
"wifi mac":"d4ab60c2fc51",
"camera type":"HERO8 Black",
,"camera serial number":"C3xxxxxxxxxxxx",
}

Thank you!

**Reply by dnewman-gpsw** [reply]
That firmware is too old for an update without an authentic battery.  

In the 2.50.75 release notes:
   2.51.75 - July 2022 
     - Support for naked cameras with factory resets and firmware updates

Labs didn't have full naked support with 2.50.71.
